.travis.yml

```diff
@@ -1,24 +1,51 @@
 language: php
 
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+  - nightly
   - hhvm
- 
+
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=1.3.*
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=2.0.*@dev
```

Comment:
The FOSUserBundle versions could be added in the matrix for some of the Symfony jobs instead of adding extra jobs for them. It would save 2 jobs in the matrix